alt_e2eshark/e2e_testing/storage.py

```diff
@@ -120,6 +120,15 @@ def write_inference_input_bin_file(modelinput, modelinputbinfilename):
         f.write(bytearr)
         f.close()
 
+def load_test_txt_file(filepath):
+    with open(filepath, "r") as file:
+        contents = file.read().split()
+    return contents
+
+def load_json_dict(filepath):
+    with open(filepath) as contents:
+        loaded_dict = json.load(contents)
+    return loaded_dict
```

Comment:
```suggestion
from typing import Dict, Any

def load_test_txt_file(filepath: str) -> list[str]:
    with open(filepath, "r") as file:
        contents = file.read().split()
    return contents

def load_json_dict(filepath: str) -> Dict[str, Any]:
    with open(filepath) as contents:
        loaded_dict = json.load(contents)
    return loaded_dict
```

Reply:
might be a nit but these look useful later down the road & should get some type hints just in case